internal/dcache/rpc/server/utils.go

```diff
@@ -153,6 +153,7 @@ func GetChunkLocal(ctc context.Context, req *models.GetChunkRequest) (*models.Ge
 	// Caller must not set SenderNodeID, catch misbehaving callers.
 	common.Assert(len(req.SenderNodeID) == 0, req.SenderNodeID)
 	req.SenderNodeID = rpc.GetMyNodeUUID()
+	req.IsLocalRV = true
```

Comment:
write a good comment explaing the reason why we do this and why we cannot do this for the non-local case